core-examples/node-js/collections/README.md

```diff
@@ -1,33 +1,27 @@
 # NodeJS SDK Subscription example
 
-In this example we use multisig wallet contract to send tokens from one wallet to another. You will learn how to
-- query GraphQL API version
-- query account balance
-- wait for appearance of a message in blockchain
-- perform aggregation query
-- perform a batched query (multiple queries in one request)
+You will learn how to
+- query accounts with filters and ordering
```

Comment:
Let's add the new sample with this scenario - which will use collections - we will demonstrate how to use them for analytics